nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -948,7 +935,37 @@ def permuted_ols(
     return outputs
 
 
-def _check_inputs_permuted_ols(n_jobs, tfce, masker, threshold, target_vars):
+def _make_array_contiguous(array):
+    """Make arrays contiguous for code efficiency."""
+    if not array.flags["C_CONTIGUOUS"]:
+        # useful to developer
+        warnings.warn("Target variates not C_CONTIGUOUS.")
```

Comment:
Codecov is complaining about testing this warning

Reply:
Yeah I don't think I will bother with this one for now